crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -566,6 +568,19 @@ impl ExecutorState {
 
         should_run &= system_conditions_met;
 
+        // SAFETY:
+        // - The caller ensures that `world` has permission to read any data
+        //   required by the system.
+        // - `update_archetype_component_access` has been called for system.
```

Comment:
This SAFETY comment of this function only mentions run conditions, it should be updated to reflect this requirement.